src/Microsoft.CodeQuality.Analyzers/CSharp/ApiDesignGuidelines/CSharpStaticHolderTypes.Fixer.cs

```diff
@@ -44,7 +44,7 @@ public class CSharpStaticHolderTypesFixer : CodeFixProvider
             }
         }
 
-        private async Task<Document> MakeClassStatic(Document document, ClassDeclarationSyntax classDeclaration, CancellationToken ct)
+        private static async Task<Document> MakeClassStatic(Document document, ClassDeclarationSyntax classDeclaration, CancellationToken ct)
```

Comment:
which rule is triggering this static changes ?

Reply:
Is it CA1822 ?

Reply:
Yes.